docs/must-gather.md

```
@@ -0,0 +1,318 @@
+# Must-gather tools
+
+Offline tools for querying must-gather archives via [`omc`](https://github.com/gmeghnag/omc). Available with `--mode offline` or `--mode dual`.
+
+See also: [User guide](user-guide.md) (including [which tools share common parameters](user-guide.md#common-parameters))
+
+### Prerequisites
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)